src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -169,6 +169,7 @@ public function get_grid_days( $date = null, $force = false ) {
 
 		// @todo remove this when the Event_Period repository is solid and clean up.
 		$using_period_repository = tribe_events_view_v2_use_period_repository();
+		$use_site_timezone = Timezones::is_mode( 'site' );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Equals sign not aligned with surrounding assignments; expected 7 spaces but found 1 space

<sub>hash: 04a687819cc844f39dced447d33fae18</sub>